2016-05/may-05.md

```diff
@@ -0,0 +1,74 @@
+## May 5 ([discuss](https://github.com/reactjs/core-notes/pull/13))
+
+### Attendees
+
+* [Ben](https://twitter.com/soprano) (React)
+* [Dan](https://twitter.com/dan_abramov) (React)
+* [Jim](http://github.com/jimfb) (React)
+* [Paul](https://twitter.com/zpao) (React)
+* [Sebastian](https://twitter.com/sebmarkbage) (React)
+* [Shayne](https://github.com/shayne) (React Native)
+* [Tom](https://twitter.com/tomocchino) (React)
+
+### ReactPerf Rewrite
+
+* [Dan](https://twitter.com/dan_abramov) got his rewrite of `ReactPerf` [merged into React master](https://github.com/facebook/react/pull/6647).
+* It is mostly functionally identical to the old one but is written in a more maintainable way.
+* At Facebook, we don’t use the console `ReactPerf` API much so we’ll need community help testing this.
+* There is a corresponding [React Native PR](https://github.com/facebook/react-native/pull/7283) that depends on these changes.
+* The change removes `ReactPerf.measure()` wrappers and instead emits events (e.g. `onBeginLifeCycleTimer`).
+* Related PRs: [#6549](https://github.com/facebook/react/pull/6549), [#6612](https://github.com/facebook/react/pull/6612), [#6633](https://github.com/facebook/react/pull/6633), [#6046](https://github.com/facebook/react/pull/6046).
+* It will most likely go into `15.1.0-alpha` early next week.
+
+### TestUtils and Enzyme
+
+* Airbnb’s [Enzyme](https://github.com/airbnb/enzyme) is well maintained and supported.
+* Our [`TestUtils`](https://facebook.github.io/react/docs/test-utils.html) are not in a great shape, and most people prefer Enzyme.
+* Should Enzyme become the official `TestUtils`?
```

Comment:
It would mean that Enzyme is linked to in the documentation and we tell people to use it rather than to `scryRenderedComponentsOfType`. For us, it would also mean no longer shipping the `TestUtils` as we don’t use them that much at Facebook.

Whether either of this is something we want to do, or not, is unclear.